File: Pascal/1036/1036.pas
var
	a, b, c : double;
	sqrt_root : double;
	r1, r2 : double;
begin
	readln(a, b, c);
	
	sqrt_root := (sqr(b) - (4*a*c));

	if (a = 0) or (sqrt_root < 0) then
		begin
			writeln('Impossivel calcular');
		end
	else
		begin
			r1 := ((-b + Sqrt(sqrt_root)) / (2*a));
			r2 := ((-b - Sqrt(sqrt_root)) / (2*a));
			
			writeln('R1 = ', r1:5:5);
			writeln('R2 = ', r2:5:5);
		end;
end.

File: C++/1036/1036.cpp
#include <cmath>
#include <cstdio>
#include <iostream>
using namespace std;

int main() {
		double a, b, c;
		double r1, r2;

		cin >> a >> b >> c;

		double root = b*b - 4*a*c;

		if (root == 0 || root < 0 || a == 0) {
				printf("Impossivel calcular\n");
		} 
		else {
				r1 = (-b + (sqrt(root))) / (2*a);
				r2 = (-b - (sqrt(root))) / (2*a);

				printf("R1 = %.5lf\nR2 = %.5lf\n", r1, r2);
	
		}
}
